docs/src/gallery.md

```diff
@@ -82,6 +82,24 @@ Style = :alpha
 Canonical = false
 ```
 
+Note that the `:alpha` style is able to automatically disambiguate labels:
+
+```@bibliography
+Pages = []
+Style = :alpha
+Canonical = false
+
+GraceJMO2007
+GraceJPB2007
+```
+
+This is achieved by automatically upgrading `style=:alpha` to the internal
```

Comment:
this reads a bit strange, since I can not directly see, whether I as a user have to do anything here?
Also the docs of the `AlphaStyle` mentions something with smart and stupid, where I (as an end user) am not sure I have to do something?

Reply:
Should be fixed now: https://juliadocs.org/DocumenterCitations.jl/previews/PR32/gallery/#alphabetic_style